poetry.lock

```diff
@@ -1,3 +1,37 @@
+[[package]]
```

Comment:
This upgraded some things that don't seem necessary. Did you use `poetry update stellar-sdk` ?

Reply:
No, I added this package by the following command

```
> poetry add "stellar-sdk>=4.0.0,<5.0.0" 

Updating dependencies
Resolving dependencies... (0.6s)

Writing lock file

Package operations: 8 installs, 8 updates, 0 removals

  - Updating idna (2.10 -> 3.2)
  - Installing multidict (5.1.0)
  - Updating typing-extensions (3.7.4.3 -> 3.10.0.0)
  - Installing async-timeout (3.0.1)
  - Updating attrs (20.3.0 -> 21.2.0)
  - Updating certifi (2020.12.5 -> 2021.5.30)
  - Updating urllib3 (1.26.3 -> 1.26.6)
  - Installing yarl (1.6.3)
  - Installing aiohttp (3.7.4.post0)
  - Updating cffi (1.14.4 -> 1.14.6)
  - Updating requests (2.25.1 -> 2.26.0)
  - Updating six (1.15.0 -> 1.16.0)
  - Installing aiohttp-sse-client (0.2.1)
  - Installing pynacl (1.4.0)
  - Installing stellar-base-sseclient (0.0.21)
  - Installing stellar-sdk (4.1.0)

```